web/packages/teleterm/src/ui/Vnet/vnetContext.tsx

```diff
@@ -55,13 +58,13 @@ export type VnetContext = {
    * Describes whether the given OS can run VNet diagnostics.
    */
   isDiagSupported: boolean;
-  status: VnetStatus;
+  state: VnetState;
   start: () => Promise<[void, Error]>;
   startAttempt: Attempt<void>;
   stop: () => Promise<[void, Error]>;
   stopAttempt: Attempt<void>;
-  listDNSZones: () => Promise<[string[], Error]>;
-  listDNSZonesAttempt: Attempt<string[]>;
+  fetchStatus: () => Promise<[StatusResponse, Error]>;
+  statusAttempt: Attempt<StatusResponse>;
```

Comment:
@ravicious let me know if calling the new endpoint `status` and then renaming the existing status field to `state` was bad

Reply:
Yeah I realized halfway through the renaming that `state` was a very overloaded term 😅 I'll think of something and switch it up

Reply:
reverted the VnetState change, not sure I love the new RPC name i picked but it works

Reply:
you caught me :)